tasks/tangd-custom-port.yml

```diff
@@ -0,0 +1,59 @@
+---
+# This task tells SELinux for the port that we want the tangd
+# service to use when distribution Fedora
+- name: Allow the custom port for tangd_port_t in SELinux
+  import_role:
+    name: linux-system-roles.selinux
+  vars:
+    selinux_ports:
+      - ports: "{{ nbde_server_port }}"
+        proto: tcp
+        setype: tangd_port_t
+        state: present
+
+# This block creates the override file for systemd with the new
+# port that we have requested
+- name: Create override file
+  block:
+    # This task checks if the directory /etc/systemd/system/tangd.socket.d
+    # exist and registers the value in systemd__system__tangd_socket
+    - name: Check if directory /etc/systemd/system/tangd.socket.d exist
+      stat:
+        path: /etc/systemd/system/tangd.socket.d
+      register: systemd_system_tangd_socket
+
+    # This tasks Create the /etc/systemd/system/tangd.socket.d directory if
+    # it does not exist
+    - name: Create a directory if it does not exist
+      file:
+        path: /etc/systemd/system/tangd.socket.d
+        state: directory
+        mode: '0755'
+      when: systemd_system_tangd_socket.stat.exists
+
+    # This tasks creates the file with the port entry that we want tangd to
+    # listen to
+    - name: Creates the file with the port entry that we want tangd to listen to
+      template:
+        src: tangd_socket_override.conf.j2
+        dest: /etc/systemd/system/tangd.socket.d/override.conf
+        backup: true
+        mode: '0644'
+      register: __nbde_server_daemon_reload
+
+    # This tasks reload the daemons so the new changes take effect
+    - name: Reload the daemons so the new changes take effect
+      systemd:
+        daemon_reload: true
+      when: __nbde_server_daemon_reload is changed
+
+- name: Ensure the desired port is added to firewalld
+  import_role:
+    name: linux-system-roles.firewall
```

Comment:
`name: fedora.linux_system_roles.firewall`

Reply:
I've changed the role name also in the test file. Hope it was correct

Reply:
Now the roles are working without installing the linux-system-roles package. Now I get why my role was not found before installing the package. So the issue in firewall system role can be closed